features/step_definitions/helpers.rb

```diff
@@ -64,7 +64,7 @@ def initialize_task_params(project, story, user = User.first)
   params
 end
 
-def initialize_work_package_params(project, type = Type.first, parent = nil, user = User.find(:first))
+def initialize_work_package_params(project, type = Type.first, parent = nil, user = User.first)
```

Comment:
Assignment Branch Condition size for initialize_work_package_params is too high. [16.06/15]<br>Line is too long. [95/80]